stable/fluentd/values.yaml

```diff
@@ -167,3 +167,22 @@ nodeSelector: {}
 tolerations: []
 
 affinity: {}
+
+# Enable autoscaling using HorizontalPodAutoscaler
+autoscaling:
+  enabled: false
+  minReplicas: 2
+  maxReplicas: 5
+  metrics:
+    - type: Resource
+      resource:
+        name: cpu
+        target:
+          type: Utilization
+          averageUtilization: 60
```

Comment:
these settings are a bit opinionated or focused on a specific threshold. All these settings should be configurable as well.

Reply:
These are only default, these are totally configurable.
The chart consume the entire "metrics" field  AS-IS

Reply:
okay, it sounds good to me. I missed this structure somehow.